svc-bie-kafka/src/main/java/gov/va/vro/services/bie/service/kafka/KafkaConsumer.java

```diff
@@ -0,0 +1,44 @@
+package gov.va.vro.services.bie.service.kafka;
+
+import gov.va.vro.services.bie.config.BieProperties;
+import gov.va.vro.services.bie.service.AmqpMessageSender;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.apache.kafka.clients.consumer.ConsumerRecord;
+import org.springframework.kafka.annotation.KafkaListener;
+import org.springframework.stereotype.Component;
+
+import java.nio.charset.StandardCharsets;
+
+@Slf4j
+@RequiredArgsConstructor
+@Component
+public class KafkaConsumer {
+  private final AmqpMessageSender amqpMessageSender;
+  private final BieProperties bieProperties;
+
+  @KafkaListener(
+      topics = {
+        "#{'${kafka.topic.prefix}'}_CONTENTION_BIE_CONTENTION_ASSOCIATED_TO_CLAIM_V02",
+        "#{'${kafka.topic.prefix}'}_CONTENTION_BIE_CONTENTION_UPDATED_V02",
+        "#{'${kafka.topic.prefix}'}_CONTENTION_BIE_CONTENTION_CLASSIFIED_V02",
+        "#{'${kafka.topic.prefix}'}_CONTENTION_BIE_CONTENTION_COMPLETED_V02",
+        "#{'${kafka.topic.prefix}'}_CONTENTION_BIE_CONTENTION_DELETED_V02"
+      })
+  public void consume(ConsumerRecord<byte[], byte[]> record) {
+    try {
+      String messageKey = new String(record.key(), StandardCharsets.UTF_8);
+      String messageValue = new String(record.value(), StandardCharsets.UTF_8);
+      String topicName = record.topic();
+
+      log.info("Topic name: {}", topicName);
+      log.info("Consumed message key: {}", messageKey);
+      log.info("Consumed message value (before) decode: {}", messageValue);
+
+      amqpMessageSender.send(
+          bieProperties.getKafkaTopicToAmqpExchangeMap().get(topicName), topicName, messageValue);
+    } catch (Exception e) {
```

Comment:
Logging of  exceptions should be done by the service's exception-handling module not the business logic.  Single responsiblity and cohesion imply so.  I would just let exceptions go in this case.

Reply:
The try catch was for schema registry decoding kafka message using schema file, the code is moving to another PR, so I clean up the try catch here.